addon/components/imgix-image.js

```diff
@@ -73,34 +51,10 @@ export default Component.extend(ResizeAware, {
       this._handleImageError = this._handleImageError.bind(this);
       this.element.addEventListener('error', this._handleImageError);
     }
-
-    this.didResize(
-      get(this, 'width') ||
-        get(this, '_width') ||
-        this.element.clientWidth ||
-        this.element.parentElement.clientWidth,
-      get(this, 'height') ||
-        get(this, '_height') ||
-        this.element.clientHeight ||
-        this.element.parentElement.clientHeight
-    );
   },
 
   didUpdateAttrs(...args) {
```

Comment:
If all this is gonna do is call `super`, can we remove the definition entirely?

Reply:
@Duder-onomy are we able to do this?

Reply:
Yeah, we can eliminate that hook.